src/components/NavBar/NavBar.js

```diff
@@ -18,6 +22,12 @@ export const NavBar = ({ title, logo }) => {
             <button className="btn btn-primary">Login</button>
           </Link>
         </li>
+        <li className="nav-action-item">
+          <i
+            class={`far ${theme === "dark" ? "fa-sun" : "fa-moon"} fa-2x`}
+            onClick={() => setTheme(theme === "dark" ? "theme" : "dark")}
```

Comment:
If the condition ```theme === 'dark``` does not meet, theme should be set to 'light'. Code will work for anything other than 'dark', but for consistency and readability you may consider changing it to 'light'.
Hope I am not missing anything here.

Reply:
Hey Tanay! I did considered this while implementing the feature but did not go with it, as I thought it would be a bit misdirecting as there is no definition of "light" class anywhere in styles.